src/hooks/use-menu/__tests__/index.test.tsx

```diff
@@ -0,0 +1,53 @@
+import { renderHook } from '@testing-library/react-hooks'
+import { useMenu } from '..'
+
+describe('useMenu', () => {
+  it('getTriggerProps should apply aria and class attributes correctly', () => {
+    const { result } = renderHook(() => useMenu())
+    const triggerProps = result.current.getTriggerProps({
+      className: 'custom-class',
+    })
+
+    expect(triggerProps['aria-haspopup']).toBe(true)
+    expect(triggerProps['aria-expanded']).toBe(false)
+  })
+
+  it('getPopoverProps should apply data-open attribute correctly', () => {
+    const { result } = renderHook(() => useMenu())
+
+    const popoverProps = result.current.getPopoverProps()
+    expect(popoverProps['data-open']).toBe(false)
+
+    result.current.openMenu()
```

Comment:
**question:** should we exercise `openMenu` in the `getTriggerProps` as well in order to ensure `aria-expanded` becomes true?

Reply:
yes sure, `getTriggerProps` already have test for its `onClick` just need to add one assertion.